woocommerce/woocommerce
# Show community forum link if wccom is not connected

### All Submissions:






### Changes proposed in this Pull Request:

When a store is not connected to WooCommerce.com, showing the a link to premium support is bad UX. This PR links support forum, and optionally premium support link if the store is connected to WooCommerce.com

Screenshot when the store is connected:
![Screenshot 2020-07-02 at 1 27 32 AM](https://user-images.githubusercontent.com/7571618/86285981-41051f80-bc03-11ea-8e50-2a2dacd275d4.png)


Screenshot when the store is not connected:
![Screenshot 2020-07-02 at 1 21 03 AM](https://user-images.githubusercontent.com/7571618/86285938-2df24f80-bc03-11ea-8a61-7505da05ad2d.png)




Closes #25442.

### How to test the changes in this Pull Request:

1. Connect store to WooCommerce.com
2. Go to the `wp-admin/plugins.php` page.
3. You should be able to see both support forum and premium support forum links.
4. Disconnect store from WooCommerce.com (Go to WooCommerce > Extensions > WooCommerce.com Subscriptions, then from `Connected to WooCommerce.com` dropbox, select disconnect.)
5. Go back to the `wp-admin/plugins.php` page.
6. Premium support link would not show.

### Other information:




### Changelog entry

> Fix - Don't show premium support forum link if the store is not connected to WooCommerce.com.
